src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Cmdlets/GetPackageCommand.cs

```diff
@@ -192,7 +192,6 @@ protected override void ProcessRecordCore()
         /// <param name="packagesToDisplay"></param>
         private async Task WriteUpdatePackagesFromRemoteSourceAsync(NuGetProject project)
         {
-            var frameworks = PowerShellCmdletsUtility.GetProjectTargetFrameworks(project);
```

Comment:
Not really, `get-package` command returns all the installed packages in the solution or project so I doubt we want to search by tfm there.